seabird/irc.py

```diff
@@ -4,16 +4,18 @@
 LOG = logging.getLogger(__name__)
 
 
+# https://github.com/ircv3/ircv3-specifications/blob/master/core/message-tags-3.2.md#escaping-values
+TAG_MAPPING_VALUES = [
```

Comment:
I'd make this a dict instead since you can still call `.items()` later to get an iterable of tuples like you have here